aws-iot-device-sdk/index.d.ts

```diff
@@ -0,0 +1,405 @@
+// Type definitions for aws-iot-device-sdk 1.0.13
+// Project: https://github.com/aws/aws-iot-device-sdk-js
+// Definitions by: Markus Olsson <https://github.com/niik>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="mqtt" />
+/// <reference types="ws" />
+/// <reference types="node" />
+
+declare module "aws-iot-device-sdk" {
```

Comment:
could you strip `declare module "aws-iot-device-sdk" {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred